src/api/java/genkinds.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+#
+# genkinds.py
+# Top contributors (to current version):
+#   Mudathir Mohamed
+# This file is part of the CVC4 project.
+# Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+# in the top-level source directory and their institutional affiliations.
+# All rights reserved.  See the file COPYING in the top-level source
+# directory for licensing information.
+#
+
+"""
+This script reads CVC4/src/api/cvc4cppkind.h and generates
+cvc/Kind.java file which declare all the CVC4 kinds.
+"""
+
+import os
+import sys
+
+# the following command in CVC4/build/src/api/java/CMakeFiles/gen-java-kinds.dir/build.make
+# cd CVC4/build/src/api/java && /usr/bin/python3 CVC4/src/api/java/genkinds.py ...
+# indicates we are in directory CVC4/build/src/api/java
+# so we use ../../../../src/api to access CVC4/src/api/parsekinds.py
+sys.path.insert(0, os.path.abspath('../../../../src/api'))
```

Comment:
This is fine for now but I would prefer to group the files together so that we don't have to resort to manipulating `sys.path`.